### DISCUSSION TITLE
Problem with social log ins in production website

### DISCUSSION THREAD

**Discussion by Parthkhandelwal0** [reown-com/appkit#3416] [Q&A]
I am having trouble with the social log ins with reown appkit, I work for intelagen and we are trying to integrate the REOWN appkit into our product, it works on local dev but when we deploy it in production the social log in does not work. Is there anyone who can help me with that?

```import { createAppKit } from "@reown/appkit/react";
import { WagmiProvider } from "wagmi";
import { base } from "@reown/appkit/networks";
import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
import { WagmiAdapter } from "@reown/appkit-adapter-wagmi";

// Setup queryClient
const queryClient = new QueryClient();

// projectId from https://cloud.reown.com (can change later)
const projectId = import.meta.env.VITE_REOWN_PROJECT_ID;
// Set the networks
const networks = [base];

// Create Wagmi Adapter
const wagmiAdapter = new WagmiAdapter({
    networks,
    projectId,
    ssr: true,
});

// Create modal
createAppKit({
    adapters: [wagmiAdapter],
    featuredWalletIds: [
        "fd20dc426fb37566d803205b19bbc1d4096b248ac04548e3cfb6b3a38bd033aa",
    ],
    networks,
    projectId,
    themeVariables: {
        "--w3m-accent": "#000080",
    },
});

export function AppKitProvider({ children }) {
    return (
        <WagmiProvider config={wagmiAdapter.wagmiConfig}>
            <QueryClientProvider client={queryClient}>
                {children}
            </QueryClientProvider>
        </WagmiProvider>
    );
}
```

This is the appkit setup

**Comment by arein** [comment]
hey @Parthkhandelwal0 - can we repro this somewhere? Do you see any error messages in the console? Do you have an allowlist set on cloud.reown.com and if so does it allow your production domain? Does it work for you on https://appkit-lab.reown.com/library/wagmi/ ?

What exact error are you seeing?

**Reply by Parthkhandelwal0** [reply]
Hey @arein I am getting this error in the console ```Refused to frame 'https://secure.walletconnect.org/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'self' http://localhost:* https://*.pages.dev https://*.vercel.app https://*.ngrok-free.app https://secure-mobile.walletconnect.com https://secure-mobile.walletconnect.org"```

**Reply by arein** [reply]
which version of appkit are you on? but probably a good idea to add your domain in the allowlist in cloud.reown.com

this error shows that whatever domain you are on is not allowlisted

**Reply by Parthkhandelwal0** [reply]
Okay thank you, these are the versions I am using ```"@reown/appkit": "^1.3.0",
    "@reown/appkit-adapter-wagmi": "^1.3.0",
    "@tanstack/react-query": "^5.59.20",
    "@web3modal/wagmi": "^5.1.11",```
    
    where can I put the domain on allow list? I could not find it in the project dashboard
